configJson.js

```diff
@@ -0,0 +1,24 @@
+const configBuilder = require('openfin-config-builder');
+
+if (process.argv.length <= 2) {
+    console.log('Usage: ' + __filename + ' <deployName>');
+    process.exit(-1);
+}
+
+const deployName = process.argv[2];
```

Comment:
Worth validating that this is either a valid semver (see https://www.npmjs.com/package/semver) or `dev` or `master`?